client/src/store/modules/simulation.ts

```diff
@@ -45,6 +47,10 @@ const getters: GetterTree<any, HMI.SimulationParameter[]> = {
   getSimVariables (state): HMI.SimulationVariable[] {
     return state.variables;
   },
+
+  hasVariablesRuns (state): boolean {
```

Comment:
hmm im not a massive fan of this, i think the default (empty) state of variables could use some work to make this long list of conditionals unnecessary

i think ideally we should change the default state so that `state.variables?.[0]?.values.length` is all thats required

i could try to do this on merge; this is more a note to self than anything else

Reply:
Exactly, I thought this could be done on a cleanup or update of the store later on, instead of fixing that up with 3 PR in progress on the simulation view.